images/TEMPLATE/main-config.tf

```diff
@@ -12,7 +12,7 @@ variable "extra_packages" {
 }
```

Comment:
Should `TODO YOUR PACKAGE HERE` be replaced with `.PackageName`? If so, we could also update the `apko` YAML to include the comment:

`# {{ .PackageName }} comes in via var.extra_packages`

Reply:
Yes that makes sense to me!

Reply:
Done!